docs/pypi.md

```diff
@@ -33,10 +33,39 @@ finished loading:
 
 ```
 def do_work(*args):
-  import snowballstemmer
-  snowballstemmer.stemmer('english')
-  stemmer.stemWords('go goes going gone'.split())
+    import snowballstemmer
+    stemmer = snowballstemmer.stemmer('english')
+    print(stemmer.stemWords('go goes going gone'.split()))
 
 import micropip
 micropip.install('snowballstemmer').then(do_work)
 ```
+
+## A Minimal working example
+
+```html
+<html>
+<head>
```

Comment:
Maybe add a `<meta charset="utf-8">` to avoid the encoding warnings in console.

Reply:
Yup, will do.